docs/learn/learn-comparison-avalanche.md

```diff
@@ -0,0 +1,104 @@
+
+#### P-chain (Platform)
+
+The P-chain is responsible for the validator set and securing the network. AVAX token holders can spin up their own nodes and become validators by staking their tokens. Similar to the NPoS system that Polkadot uses, Avalanche allows token holders who do not want to spin up their own node to become validators to delegate their token stake to existing validators. When a new subnet is spun up, it will also be connected to the p-chain. 
+
+<!-- TODO: do subnet validators also have to validate the p-chain? In Polkadot collators do not have to do this, but they do have to sync with the relay chain effectively running a relay chain node on their machine. -->
+
+#### X-chain (Exchange)
+
+The X-chain is responsible for the transaction layer of the Avalanche blockchain. It uses a UTXO model like Bitcoin vs Polkadot which uses an account model like Ethereum.
+
+<!-- TODO: Figure out what a Directed Acyclyc Graph (DAG) is in accordance to a linear blockchain. And state the different in the Polkadot model -->
+
+#### C-chain (Contracts)
+
+The C-chain is where most activity will happen on the Avalanche blockchain. It allows for different virtual machines to execute smart contract code. Out of the box it has support for EVM and AVM(Avalanche VM). 
+
+#### Subnets or sub-networks
+
+Subnets can be public or private and allow for developers to design and launch a blockchain on Avalanche. These are one or many validators that are subsets from the overall validator set of the network. Any validator in the Avalanche network has to validate all three chains making them a part of the primary network subnet. Subnets can validate any blockchains, however each blcokcahin is validated by exactly one subnet. 
+
+<!-- TODO: How different is this model from the parachain model? Do subnets also have to use the avalanche consensus model? They have a choice of VM but do they of consensus? -->
+
+## Consensus 
+
+<!-- TODO: Dig into each Snow protocol in detail -->
+
+Avalanche uses a family of consensus protocols to achieve security, liveness and finality. These are known as the Snow* protocols. This group of protocol composed together use both classical and Nakomoto consensus as well as a Delegated Proof-of-Stake system for its block creators.
+
+The Snow family is a hierarchical collection of systems used to reach finality on Avalanche:
+
+- Slush
+- Snowflake
+- Snowball
+
+
+Compared to Polkadot, Avalanche uses a asynchronous hybrid system that is based on a clasical and Nakomoto approach. Polkadot uses an synchronous hybrid model that combines BABE and GRADNPA, where BABE is the algorithm used to build blocks in a probabilistic way, and GRANDPA is a finality mechanism that uses a deterministic approach to adding blocks to the longest chain. In the end, validators agree to whole chains, rather than single new blocks.
```

Comment:
```suggestion
Compared to Polkadot, Avalanche uses a asynchronous hybrid system that is based on a clasical and Nakomoto approach. Polkadot uses a synchronous hybrid model that combines BABE and GRADNPA, where BABE is the algorithm used to build blocks in a probabilistic way, and GRANDPA is a finality mechanism that uses a deterministic approach to adding blocks to the longest chain. In the end, validators agree to whole chains, rather than single new blocks.
```